src/main/kotlin/net/atos/client/smartdocuments/model/wizard/SmartDocumentsResponseClasses.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.client.smartdocuments.model.wizard
+
+import jakarta.json.bind.annotation.JsonbProperty
+import nl.lifely.zac.util.NoArgConstructor
+
+@NoArgConstructor
+data class UnattendedResponse(
+    @field:JsonbProperty("file")
+    var files: List<File>? = null
+)
+
+@NoArgConstructor
+data class File(
+    var fileName: String? = null,
+    var document: Document? = null
+)
+
+class Document
```

Comment:
@hsiliev do you maybe know what is up with this empty class? I also saw one in `TemplatesDeclarations.kt`:

```class HeaderGroup```

Surely these should be primitive types or have member fields?

Reply:
For `HeaderGroup` we know only the field name, but not the content/structure.

We might as well just remove these, as we don't seem to care about them anyway.

Reply:
I think it's because the SmartDocuments APIs are not really documented right? I will ask them about this.

Reply:
Actually about Document, if I look at https://sandbox.smartdocuments.com/?mode=REST a `document` contains a `data` element with the actual file contents. Not sure if it's base64 encoded or something. Also looks like we are missing the `outputFormat` field. Will add this in this PR but might need some tweaking later still.

Reply:
I have added these extra fields now. For now just as strings.